core-services/release-controller/_releases/release-ocp-4.6.json

```diff
@@ -1,14 +1,18 @@
 {
-  "hide": true,
   "name":"4.6.0-0.nightly",
   "to": "release",
   "message": "This release contains OSBS official image builds of all code in release-4.6 (master) branches, and is updated after those builds are synced to quay.io.",
   "mirrorPrefix": "4.6-art-latest",
-  "expires":"12h",
-  "maxUnreadyReleases": 1,
-  "minCreationIntervalSeconds": 7200,
+  "expires":"168h",
+  "maxUnreadyReleases": 2,
+  "minCreationIntervalSeconds": 2400,
   "referenceMode": "source",
   "pullSecretName": "source",
+  "check":{
+￼   "OCP and Origin images should match": {
```

Comment:
you have some kind of weird prefix char at the start of this line.

Reply:
Yeah, I have no idea how that got there!  It's gone now!